packages/svelte-fast-dimension/src/index.ts

```diff
@@ -25,7 +35,7 @@ export function fastDimension(): PreprocessorGroup {
 
 						const expressions = elementToCompiledExpressions.get(parent) as string[];
 
-						expressions.push(`${node.expression.name} = e.target.${node.name}`);
+						expressions.push(`${expressionToString(node.expression)} = e.target.${node.name}`);
```

Comment:
Silly me for using `node.expression.name`. For this case, we should slice the string instead of extracting deeper down the AST so it's more flexible. Something like:

```js
const boundVar = s.slice(node.expression.start, node.expression.end)
expressions.push(`${boundVar} = e.target.${node.name}`);
```

Reply:
ok thanks for the recommendation, fixed.